test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -146,14 +146,19 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        //[LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        [Theory, VaryingTolerance(4)]
```

Comment:
This is using the same tolerance as before. Maybe this test doesn't need the VaryingTolerance attribute. Please try enabling the test in CI builds without this attribute first.